CONTRIBUTING.md

```diff
@@ -1,63 +1,59 @@
-Contributing to Flutter
-=======================
+# Contributing to Flutter
```

Comment:
I'm not sure this is prettier... if someone is reading this in their text editor (which seems at least as likely as reading it on the Web site) it's a lot clearer that something is a header if it's underlined.